widget/playground/src/constants/presets.ts

```diff
@@ -42,20 +42,40 @@ export const PRESETS: {
       neutral: '#E6E6E6',
       background: '#FDFDFD',
       foreground: '#010101',
+      info: '#5BABFF',
     },
   },
   {
     id: 2,
+    dark: {
+      neutral: '#434965',
+      primary: '#1C3CF1',
+      secondary: '#2284ED',
+      background: '#070917',
+      foreground: '#FDFDFD',
+      info: '#5BABFF',
+    },
     light: {
```

Comment:
I two last light themes, background colors is same as widget top background color. is there any way to fix this?
![Recording 2023-11-18 at 14 01 19](https://github.com/rango-exchange/rango-client/assets/99180934/79c4ee29-d43e-4ba1-94d6-6b4c372c7466)

Reply:
fixed